spec/integ/matrix-client-retrying.spec.js

```diff
@@ -93,7 +93,14 @@ describe("MatrixClient retrying", function() {
             }).toThrow();
         }).respond(400); // fail the first message
 
-        const p3 = httpBackend.flush().then(function() {
+        // wait for the localecho of ev1 to be updated
+        const p3 = new Promise((resolve, reject) => {
+            room.on("Room.localEchoUpdated", (ev0) => {
+                if(ev0 === ev1) {
+                    resolve();
+                }
```

Comment:
no, the thing is that there are a couple of localEchoUpdated events and we specifically want to wait for the ev1 one.